svo (actions dictionary).xml

```diff
@@ -13167,24 +13167,14 @@ if svo.me.class == 'Druid' then
 end
 if svo.haveskillset('kaido') then
   svo.dict.transmute = {
-    -- transmutespam is used to throttle bleed spamming so it doesn't get out of control
-    transmutespam = false,
-    transmutereps = 0,
     physical = {
       balanceless_act = true,
 
       isadvisable = function()
-        return (conf.transmute ~= 'none' and not defc.dragonform and (stats.currenthealth &lt; sys.transmuteamount or (sk.gettingfullstats and stats.currenthealth &lt; stats.maxhealth)) and not svo.doingaction'healhealth' and not svo.doingaction'transmute' and not codepaste.balanceful_codepaste() and svo.can_usemana() and (not affs.prone or svo.doingaction'prone') and not svo.dict.transmute.transmutespam) or false
+        return (conf.transmute ~= 'none' and not defc.dragonform and (stats.currenthealth &lt; sys.transmuteamount or (sk.gettingfullstats and stats.currenthealth &lt; stats.maxhealth)) and not svo.doingaction'healhealth' and not svo.doingaction'transmute' and not codepaste.balanceful_codepaste() and svo.can_usemana() and (not affs.prone or svo.doingaction'prone')) or false
       end,
 
       oncompleted = function()
```

Comment:
Just remove it entirely

Reply:
Just remove transmutespam entirely? I thought I did. Or are you saying to remove the dict entry for transmute?

Reply:
It looked like you left an empty `oncompleted` function in?

Reply:
Ah, missed it